learn/ai_powered_search/rest_response_request.mdx

```diff
@@ -155,18 +174,51 @@ Cloudflare's embeddings are instead located in `response.result.data`:
 }
 ```
 
+<Capsule intent="note">
+The prompt data type does not necessarily match the response data type. For example, Cloudflare always returns an array of embeddings, even if the prompt in your request was a string.
+</Capsule>
+
 Meilisearch silently ignores fields not pointing to an `{{embedding}}` value.
 
+## The embedding header
+
+Your provider might also request you to add specific headers to your request. For example, Azure's AI services require a `api-key` header containing an API key.
+
+Add the `headers` field to your embedder object:
+
+```json
+{
+  "EMBEDDER_NAME": {
+    "source": "rest",
+    "url": "PROVIDER_URL",
+    "request": {
+      "text": "{{text}}"
+    },
+    "response": {
+      "result": {
+        "data": "[{{embedding}}]"
+      }
+    },
+    "header": {
+      "FIELD_NAME": "FIELD_VALUE"
+    }
+  }
+}
+```
+
+By default, Meilisearch includes a `Content-Type` header. It may also include an authorization bearer token, if you have supplied an API key.
```

Comment:
Should we specify the value of the Content-Type header?

- `Content-Type: application/json`

Reply:
I don't think it's super necessary. The point of the guide is to show people how to use `request`/`response`/`headers` to match the requirements from their provider. It's relevant that people know there are two default headers, but they can consult the reference if they need exhaustive information.